src/components/TextTimer/TextTimer.js

```diff
@@ -0,0 +1,21 @@
+import './TextTimer.css';
+
+export default function TextTimer(minute, sec) {
+
+    const counterOutput = document.createElement('div');
+    counterOutput.className = 'counterOutput';
+    let remainingTime = `${minute} : ${sec}`;
+
+    if (minute === 0 && sec === 0) {
+        remainingTime = 'Time end';
+    } else {
+        if (sec < 10) {
+            sec = `0${sec}`;
+        }
+        remainingTime = `Time Left: ${minute}m : ${sec}s`;
```

Comment:
remove this `if (sec < 10)` and do helper function which prepare proper way to display seconds

helper function should look like that ```formatSeconds = (seconds) => `0$seconds```

and then remaining time should look like 
```
remianingTime = `time left: $minute : $sec < 10 ? formatSeconds(seconds) : sec`s 
```